agent/app/agent_capability_windows.go

```diff
@@ -142,3 +143,7 @@ func defaultIsPlatformExecSupported() (bool, error) {
 	}
 	return true, nil
 }
+
+func defaultIsGuardDutySupported() (bool, error) {
+	return false, errors.New("GuardDuty is currently not supported on windows")
```

Comment:
return `false, nil` like in `agent_capability_unspecified.go`? This function is a no-op for unsupported platforms, rather than returning error by default

Reply:
Updated this